internal/service/advancedcluster/resource_advanced_cluster_migration_test.go

```diff
@@ -12,135 +12,85 @@ import (
-		PreCheck:     func() { mig.PreCheckBasic(t) },
-		CheckDestroy: acc.CheckDestroyCluster,
-		Steps: []resource.TestStep{
-			{
-				ExternalProviders: mig.ExternalProviders(),
-				Config:            config,
-				Check:             checkShardedMultiCloud(clusterName, 1, "M30", false),
-			},
-			mig.TestStepCheckEmptyPlan(config),
-			{
-				ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
-				Config:                   config,
-				Check:                    checkShardedMultiCloud(clusterName, 1, "M30", true), // external_id will be present in latest version
-			},
-		},
-	})
+func TestMigAdvancedCluster_symmetricGeoShardedOldSchema(t *testing.T) {
+	// once 1.18.0 is released we can adjust this to always check new attributes - CLOUDP-266096
+	testCase := symmetricGeoShardedOldSchemaTestCase(t, false)
+	mig.CreateAndRunTest(t, &testCase)
+}
+
+func TestMigAdvancedCluster_asymmetricShardedNewSchema(t *testing.T) {
+	acc.SkipTestForCI(t) // latest release does not support ISS, to be adjusted in CLOUDP-266096
+	testCase := asymmetricShardedNewSchemaTestCase(t)
+	mig.CreateAndRunTest(t, &testCase)
 }
 
-func TestMigAdvancedCluster_singleShardPerZoneGeoSharded(t *testing.T) {
+func TestMigAdvancedCluster_replicaSetAWSProviderUpdateAfterVerisonUpgrade(t *testing.T) {
 	var (
-		orgID       = os.Getenv("MONGODB_ATLAS_ORG_ID")
-		projectName = acc.RandomProjectName() // No ProjectIDExecution to avoid cross-region limits because multi-region
+		projectID   = acc.ProjectIDExecution(t)
 		clusterName = acc.RandomClusterName()
-		config      = configGeoShardedOldSchema(orgID, projectName, clusterName, 1, 1, false)
 	)
 
 	resource.ParallelTest(t, resource.TestCase{
 		PreCheck:     func() { mig.PreCheckBasic(t) },
 		CheckDestroy: acc.CheckDestroyCluster,
 		Steps: []resource.TestStep{
 			{
-				ExternalProviders: mig.ExternalProviders(),
-				Config:            config,
-				Check:             checkGeoShardedOldSchema(clusterName, 1, 1, false, false),
+				ExternalProviders: acc.ExternalProviders("1.17.5"), // last version that did not support new sharding schema or attributes
+				Config:            configReplicaSetAWSProvider(projectID, clusterName, 60, 3),
+				Check:             checkReplicaSetAWSProvider(projectID, clusterName, 60, 3, false, false),
 			},
-			mig.TestStepCheckEmptyPlan(config),
 			{
 				ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
-				Config:                   config,
-				Check:                    checkGeoShardedOldSchema(clusterName, 1, 1, true, true), // external_id will be present in latest version
+				Config:                   configReplicaSetAWSProvider(projectID, clusterName, 60, 5),
+				Check:                    checkReplicaSetAWSProvider(projectID, clusterName, 60, 5, true, true),
 			},
 		},
 	})
 }
 
-func TestMigAdvancedCluster_symmetricGeoShardedOldSchema(t *testing.T) {
+func TestMigAdvancedCluster_geoShardedOldSchemaUpdateAfterVerisonUpgrade(t *testing.T) {
```

Comment:
```suggestion
func TestMigAdvancedCluster_geoShardedOldSchemaUpdate(t *testing.T) {
```

Reply:
since these are migration tests (which are used for comparison between version upgrades maybe we can exclude that from the name)